src/common/packager.ts

```diff
@@ -191,28 +195,36 @@ export class Packager {
                     executedStartPackagerCmd = true;
                     return this.monkeyPatchOpnForRNPackager()
                         .then(() => {
-                            let args = ["--port", port.toString()];
+                            let args: any = ["--port", port.toString()];
```

Comment:
_It shouldn't contain anything other than strings_ - semantically `args` is an array of parameters passed to the packages and commandline parameters are always strings.

The line that you pointing me to (`args.push("--assetExts", ["ttf"])`) is also very confusing and probably would look better if rewritten to `args.push("--assetExts", ["ttf"].toString())`

Another argument is that I perfectly remember that you've had a lot of troubles you could avoid if array value was converted to string explicitly